WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridFragment.java

```diff
@@ -468,6 +468,7 @@ public void onMediaListFetched(OnMediaListFetched event) {
     void reload() {
         if (isAdded()) {
             getAdapter().setMediaList(getFilteredMedia());
+            setFilter(mFilter);
```

Comment:
The `setFiler` method contains the `getAdapter().setMediaList(getFilteredMedia())` statement so we can remove it from the `reload` method to avoid duplicate calls.

Reply:
After changing RecyclerView to EmptyViewRecyclerView I deleted the `setFilter` method